myuw/templates/handlebars/card/accounts/tuition.html

```diff
@@ -2,42 +2,44 @@
     {% tplhandlebars "tuition_card" %}
 
 <div class="card"  data-name="TuitionCard" data-type="card">
-
-    {{#if is_pce}}
+	<!-- Card title -->
+    {{ #if is_pce}}
         <h3>PCE Account Balance</h3>
-    {{else}}
+    {{ else }}
         <h3>Tuition &amp; Fees</h3>
-    {{/if}}
+    {{ /if }}
 
-    {{#if due_today}}
         <!-- if tuition due date is TODAY -->
+    {{ #if due_today}}
         <div class="card-critical-alert">
-            <i class="fa fa-exclamation-circle"></i> {{#if is_pce}} PCE tuition{{else}}Tuition{{/if}} is due today.
+            <i class="fa fa-exclamation-circle"></i> {{ #if is_pce}} PCE tuition{{ else  }}Tuition{{ /if }} is due today.
         </div>
+    {{ /if }}
         <!-- end if -->
-    {{/if}}
 
-    {{#if past_due}}
         <!-- if tuition due date has passed and balance is greater than 0 -->
+    {{ #if past_due }}
         <div class="card-critical-alert">
-            <i class="fa fa-exclamation-circle" aria-hidden="true"></i> {{#if is_pce}} Your PCE tuition is past due.{{else}}You have a balance that may be past due. {{/if}}<a href="https://sdb.admin.uw.edu/sisStudents/uwnetid/tuition.aspx" target="_blank" data-linklabel="Tuition Statement">See your statement for details</a>.
+            <i class="fa fa-exclamation-circle" aria-hidden="true"></i> {{ #if is_pce}} Your PCE tuition is past due.{{ else  }}You have a balance that may be past due. {{ /if }}<a href="https://sdb.admin.uw.edu/sisStudents/uwnetid/tuition.aspx" target="_blank" data-linklabel="Tuition Statement">See your statement for details</a>.
         </div>
+    {{ /if }}
         <!-- end if -->
-    {{/if}}
 
-    {{#unless has_balance}}
+    {{ #unless has_balance }}
         <!-- if tuition balance is 0 -->
-        <div class="card-badge-container">
+		<div class="card-badge-container ">
             <div class="card-badge">
-                <span class="card-badge-label" style="line-height:20px;">{{#if is_pce}}Your PCE tuition<br/> has been paid{{else}}Your tuition and fees have been paid{{/if}}</span>
+                <span class="card-badge-label" style="line-height:20px;">{{ #if is_pce}}Your PCE tuition<br/> has been paid{{ else  }}Your tuition and fees have been paid{{ /if }}</span>
                 <div class="card-badge-action" style="margin-top:10px; text-align:right;"><a href="https://sdb.admin.uw.edu/sisStudents/uwnetid/tuition.aspx" target="_blank" data-linklabel="Tuition Statement">See tuition statement</a></div>
             </div>
         </div>
-    {{/unless}}
+    {{ /unless }}
 
-    {{#if has_balance}}
-        {{#if is_credit}}
-            <div class="card-badge-container">
+    {{ #if has_balance }}
+        <div class="card-badge-container ">
+	    <ul class="card_list">
+        {{ #if is_credit }}
+         <div class="card-badge-container">
```

Comment:
This logic and nesting of elements here needs a review. If there is a balance and credit, this will print
```
<div class="card-badge-container ">
  <ul class="card_list">
    <div class="card-badge-container">
      <ul class="card_list">
```
which is invalid.